server/apps/fieldsight/loader.py

```diff
@@ -188,15 +200,16 @@ def load_household(self, datum):
         defaults['geosite'] = geosite
 
         try:
-            defaults['district'], _ = District.objects.get_or_create(
-                name=get_attr(datum, 'District_of_origin')
-            )
-            defaults['gaupalika'], _ = Gaupalika.objects.get_or_create(
-                name=get_attr(datum, 'Gaupalika_Municipality'),
-                defaults={'district': defaults['district']}
-            )
+            defaults['district'] = District.objects.filter(
```

Comment:
We should have a threshold for trigram similarity.

Should log errors for which similarity is below threshold

Reply:
Done